meow-tutor.el

```diff
@@ -166,61 +165,58 @@
 =                    MOTIONS AND SELECTIONS                     =
 =================================================================
 
- Press \\[meow-next-word] to select forward until the next word.
- Inline hints as colored numbers will be showed and you could press
- consponding numbers to jump there.
+ Pressing \\[meow-next-word] will select everything from the cursor
+ position until the end of the current word. Numbers that show up
+ on the screen indicate a quick way to extend your selection. 
 
- Press \\[meow-kill] to delete the selection.
+ Pressing \\[meow-kill] will delete the current selection.
 
- The \\[meow-delete] key delete the character at the cursor, while
- \\[meow-kill] deletes all selected text.
+ The \\[meow-delete] key deletes the character at the cursor, while
+ \\[meow-kill] deletes all of the selected text.
 
  1. Move the cursor to the line below marked -->.
  2. Move to the beginning of a word that needs to be deleted.
- 3. Press \\[meow-next-word] to select a extra word.
+ 3. Press \\[meow-next-word] to select additional word.
  4. Press \\[meow-kill] to delete the selection.
  5. Repeat for all extra words in the line.
 
  --> This sentence pencil has vacuum extra words in the it.
      This sentence has vacuum words in it.
 
- 1. Move the cursor to the line below marked -->.
- 2. Move to the beginning of a word that needs to be deleted.
- 3. Press \\[meow-next-word] to select a extra word.
- 4. Press \\[meow-kill] to delete the selection.
- 5. Repeat for all extra words in the line.
-
- --> This sentence has a lot lot lot lot of extra words words words.
-     This sentence has a lot of uncessary words words.
-
 =================================================================
 =                        MORE ON MOTIONS                        =
 =================================================================
 
- As you saw, pressing \\[meow-next-word] moves the cursor forward until the start
- of the next word, selecting the text traversed. This is useful
- for moving around text and for selecting text to operate on.
+ Using \\[meow-next-word] is not only useful for selection, but also
+ serves as a fast way to move around the text. Below are listed some
+ other useful motions.
 
  Some common motions include:
-   \\[meow-next-word] - Move forward to before the beginning of the next word.
-   \\[meow-back-word] - Move backward to the beginning of the closest backward word.
-   \\[meow-next-symbol] - Select the next symbol, symbol means all the
+   \\[meow-next-word] - Moves the cursor forward to the end of the current word.
+   \\[meow-back-word] - Moves the cursor backward to the beginning of the current word.
+   \\[meow-next-symbol] - Moves the cursor to the start of the current symbol.
```

Comment:
Oops, looks like here should be `end` instead of `start`. Will fix it after other people review.